proto/attestation/evidence.proto

```diff
@@ -37,6 +37,7 @@ enum TeePlatform {
   TEE_PLATFORM_UNSPECIFIED = 0;
   AMD_SEV_SNP = 1;
   INTEL_TDX = 2;
+  NONE = 3;
```

Comment:
NONE may cause trouble in C++ later as it appears in the global namespace - maybe TEE_PLATFORM_NONE ? cf. yaqs/6578714921009152

Reply:
Done.